README.md

```diff
@@ -112,6 +113,17 @@ end
 grant_on ['users#create', 'users#update'], badge: 'autobiographer', temporary: true do |user|
   user.name? && user.email?
 end
+
+grant_on 'users#update', badge: 'updated_my_info' do |user, current_user:|
+  user == current_user
+end
+```
+
+## Even only using the current user
```

Comment:
Suggestion: "You may use current_user only".

Also, add a new line after the title please?